plugins/reports/counter/index.php

```diff
@@ -16,8 +16,13 @@
  *
  */
 
+// Because of the use of Namespaces, this plugin now requires PHP 5.3 or better
+if (version_compare(PHP_VERSION, '5.3.0') >= 0) {
```

Comment:
My intent here was to disable the plugin to avoid accidentally triggering fatal errors if someone was still using an obsolete version of PHP.

The [Exceptions conversation](https://github.com/pkp/ojs/pull/535/files#r44421855) would limit us to 5.1.0.

I also wanted, however, to create a reusable library which would output COUNTER XML based on structured input.  I searched unsuccessfully for an existing module or tool to transform an XSD to PHP DOMDocument calls.  I ended up writing this [COUNTER tool](https://github.com/ulsdevteam/COUNTER) instead.  I would be enthralled to have a better solution.

If we stick with this tool I wrote however, we're stuck with 5.3.0 as the minimum.  It went [end-of-life a year ago](http://php.net/eol.php), so I don't feel bad about it.